play/Utils/Router.swift

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
For routing, I suggest using [NavigationStacks](https://www.youtube.com/watch?v=GZ-hQWMjT0s) as it's the latest API for navigation.

Reply:
`Router` is just manager for the path -> View binding,
see  this, (used Navigation Stack), `path` is controlled by `Router` :
 https://github.com/dhruvd-grappus/play/blob/22405b92f340552ff1dfe20b4b917d3e2add1720/play/playApp.swift

Reply:
okay, you can refer to [this](https://www.youtube.com/watch?v=QZKA4fyJerI) for making global routing solution